lxd/containers.go

```diff
@@ -117,6 +96,52 @@ func containersPost(d *Daemon, r *http.Request) Response {
 	return &asyncResponse{run: run, containers: []string{req.Name}}
 }
 
+func createFromNone(d *Daemon, req *containerPostReq) Response {
+	dpath := shared.VarPath("lxc", req.Name)
+	if !shared.PathExists(dpath) {
+		return InternalError(fmt.Errorf("Container path doesn't exist."))
+	}
```

Comment:
Should that fail though?

It doesn't seem unreasonable to me to have someone define a bunch of containers ahead of time, then going to populate their rootfs before starting them.

So start definitely should fail but I see no reason to fail at create time.